### PR TITLE
improve creative mode

### PR DESCRIPTION
**Pull request by lublak**
make working in creative mode easier.
(Was actually a by-product of testing.)

### CODE REVIEW

**Comment by dannydjdk** [comment]
Another thing that would be nice is if you pick the panel (not on a specific component) that it gives you a copy of the panel with all of its components. Currently, doing that gets you an empty panel, and there's no good way in Creative to pick up the panel.

I can add this after pulling this unless it's something that interests you.

**Comment by lublak** [comment]
@dannydjdk
It's on my todo list :)
But I wanted to finish top first and then "just now" finish the creative. Then it's over to the redstone dust.
return null throws an nullpointer exception.

**Comment by dannydjdk** [comment]
Probably return a call to the function in the super class instead of null.

File: src/main/java/com/dannyandson/tinyredstone/blocks/PanelBlock.java [hunk 1]
```diff
@@ -283,6 +284,21 @@ else if (pos.north().equals(neighborPos))
...
+        if (te instanceof PanelTile) {
+            PanelTile panelTile = (PanelTile) te;
+            /*PanelCellPos panelCellPos = PanelCellPos.fromHitVec(pos, target.getHitVec());
+
+            if (panelCellPos != null && panelTile.cells.containsKey(panelCellPos.getIndex())) {
+                return panelCellItemMap.get(panelTile.cells.get(panelCellPos.getIndex()).getClass()).getDefaultInstance();
+            }
+            */
+        }
+        return null;
```
Comment (dannydjdk) [inline src/main/java/com/dannyandson/tinyredstone/blocks/PanelBlock.java:299]: return super.getPickBlock(state, target, world, pos, player);

**Comment by lublak** [comment]
Most likely.
It's still a work in progress. Was somehow a snippet that I had written for myself, because I was too lazy to always pick out the tiny things. But I thought hey this could also be helpful in general.

**Pull request closed without merging**